daml-assistant/test/DAML/Assistant/Tests.hs

```diff
@@ -36,14 +35,39 @@ import qualified Data.Conduit.Tar as Tar
 -- unix specific
 import System.PosixCompat.Files (createSymbolicLink)
 
+-- | Replace all environment variables for test action, then restore them.
```

Comment:
A comment mentioning the `setEnv` behavior might be useful.

Reply:
:+1: